config/routes.rb

```diff
@@ -5,4 +5,8 @@
   get "/404", to: "errors#not_found", via: :all
   get "/422", to: "errors#unprocessable_entity", via: :all
   get "/500", to: "errors#internal_server_error", via: :all
+
+  get "/registrations/:step_name", to: "registrations#new", as: "new_registration"
+  post "/registrations/:step_name", to: "registrations#create", as: "registrations"
```

Comment:
Can we remove the comment? (or transition these to a `resources` route and drop the two separate routes

Reply:
Yep